app/controllers/relationships_controller.rb

```diff
@@ -0,0 +1,17 @@
+class RelationshipsController < ApplicationController
+    
+  def create
```

Comment:
We could call these methods follow and unfollow respectively, so it is more understandable

Reply:
All changes were applied in the following commits of this same PR